pom.xml

```diff
@@ -170,6 +171,125 @@
+                                        <artifactItem>
+                                            <groupId>org.jacoco</groupId>
+                                            <artifactId>org.jacoco.ant</artifactId>
+                                            <version>${jacoco.version}</version>
+                                        </artifactItem>
+                                    </artifactItems>
+                                    <stripVersion>true</stripVersion>
+                                    <outputDirectory>${project.build.directory}/jacoco-jars</outputDirectory>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <dependencies>
+                            <dependency>
+                                <groupId>org.jacoco</groupId>
+                                <artifactId>org.jacoco.ant</artifactId>
+                                <version>${jacoco.version}</version>
+                            </dependency>
+                        </dependencies>
+                        <executions>
+                            <execution>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <inherited>false</inherited>
+                                <configuration>
+                                    <target>
+                                        <taskdef name="report" classname="org.jacoco.ant.ReportTask">
+                                            <classpath path="${project.build.directory}/jacoco-jars/org.jacoco.ant.jar"/>
+                                        </taskdef>
+                                        <echo>Creating JaCoCo for Quarkus code coverage reports</echo>
+                                        <report>
+                                            <executiondata>
+                                                <fileset dir="${basedir}">
+                                                    <include name="**/target/jacoco.exec"/>
+                                                </fileset>
+                                            </executiondata>
+                                            <structure name="JaCoCo for Quarkus">
+                                                <classfiles>
+                                                    <fileset dir="${basedir}">
+                                                        <include name="**/*999-SNAPSHOT.jar"/>
+                                                        <exclude name="**/*quarkus-cli-999-SNAPSHOT.jar"/>
```

Comment:
Any reason to exclude this one?

Reply:
JaCoCo doesn't like duplicated classes in and it throws 
`[ERROR] around Ant part ...<report>... @ 8:11 in /Users/rsvoboda/git/quarkus/target/antrun/build-main.xml: Error while analyzing devtools/aesh/target/original-quarkus-cli-999-SNAPSHOT.jar@META-INF/versions/11/org/wildfly/common/archive/JDKSpecific.class. Can't add different class with same name: org/wildfly/common/archive/JDKSpecific`